client/my-sites/patterns/components/readymade-template-details/server.tsx

```diff
@@ -0,0 +1,5 @@
+import { ReadymadeTemplateDetails } from 'calypso/my-sites/patterns/components/readymade-template-details/index';
```

Comment:
`index` is unnecessary:
```suggestion
import { ReadymadeTemplateDetails } from 'calypso/my-sites/patterns/components/readymade-template-details';
```